test/e2e/operator_test.go

```diff
@@ -181,6 +181,30 @@ func TestClusterIngressUpdate(t *testing.T) {
 		t.Fatalf("failed to get default router deployment: %v", err)
 	}
 
+	// update router namespace with a label.
+	routerNS := &corev1.Namespace{}
+	err = wait.PollImmediate(1*time.Second, 10*time.Second, func() (bool, error) {
```

Comment:
We don't need to poll here. We know router deployment succeeded in openshift-ingress namespace before this call.
cl.Get() should suffice?

Reply:
Think it should. Will change.